modules/Hoot/tools/folderTree.js

```diff
@@ -590,12 +601,12 @@ export default class FolderTree extends EventEmitter {
                 }
             }
         } else if ( data.type === 'folder' ) {
-            opts = [ ...this.folderContextMenu.slice() ]; // make copy of array to not overwrite default vals
-            opts.splice( 1, 0, {
-                title: `Modify Folder ${ data.name }`,
-                _icon: 'info',
-                click: 'modifyFolder'
-            } );
+                opts = [ ...this.folderContextMenu.slice() ]; // make copy of array to not overwrite default vals
+                opts.splice( 1, 0, {
+                    title: this.selectedNodes.length ? `Move Folders ${ this.checkMultiSelectFolder(data) }` : `Modify Folder ${ data.name }`,
+                    _icon: 'info',
+                    click: 'modifyFolder'
+                } );
```

Comment:
No need for these indents.
What's the new behavior here?
Add a comment if multiple folder selections can't be renamed whilst modifying.

Reply:
thought if multiple folders are being selected, it may make sense to only allow the use to move them, if they want to rename a folder, they will have to select them individually